frontend/src/components/pages/profile/Profile.spec.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import * as TestUtils from "react-dom/test-utils";
+import Profile from "./Profile";
+import { ProfileHandlerService } from "../../../services/profile_handler_service";
```

Comment:
Are these imports necessary?

Reply:
Noted! I ended up deleting this file because I will be writing tests for profile later, but for sake of organization I will make sure to note of only importing those libraries that each file is using directly